packages/server/src/api/controllers/view/index.js

```diff
@@ -11,18 +11,11 @@ const controller = {
     const designDoc = await db.get("_design/database")
     const response = []
 
-    for (let name in designDoc.views) {
-      if (
-        !name.startsWith("all") &&
-        name !== "by_type" &&
-        name !== "by_username" &&
-        name !== "by_automation_trigger"
-      ) {
-        response.push({
-          name,
-          ...designDoc.views[name],
-        })
-      }
+    for (let name of Object.keys(designDoc.views)) {
```

Comment:
NAB: `for (let name in designDoc.views)` works here too

Reply:
Generally I tend to stick to the `for of Object.keys()` rather than the `for in` approach because usually I'm trying to treat the object like a POJO and I'd prefer to avoid walking the prototype chain as for in does (it also creates a really annoying error in Webstorm using `for in` as its usually recommended to check the object with a `hasOwnProperty` check before using the property to confirm its not something deeply nested in the prototype chain.